### DISCUSSION TITLE
Hide Worksheet (SetSheetVisible) command not working

### DISCUSSION THREAD

**Discussion by mukul26s** [qax-os/excelize#1334] [Q&A]
It was working like 7-8 days ago but all of a sudden its now stopped working.
The default "Sheet1" always shows up even if I write the hide command i.e. 

f.SetSheetVisible("Sheet1",false)

I have tried this on windows as well as mac. Please confirm if any of you is also getting the same error and it would be great if would provide me a solution for this.

Or is there any other way to get rid of the "Sheet1" worksheet, would like to know.

**Comment by xuri** [comment]
Thanks for your feedback. Which version of Excelize are you using? Can you show us a complete, standalone example program or reproducible demo? Note that as the documentation says: A workbook must contain at least one visible worksheet. If the given worksheet has been activated, this setting will be invalidated.

**Reply by mukul26s** [reply]
Yes, you are right. I was hiding the Sheet1 before setting another sheet active. So, the command nullified. 

Now, first I am setting another sheet active using f.SetActiveSheet and then I am hiding the Sheet1 using f.SetSheetVisible("Sheet1", false). And it worked like a charm.

Thank you Xuri
